src/Sulu/Component/HttpCache/HttpCache.php

```diff
@@ -93,6 +93,6 @@ private function getReverseProxyTtl(Response $response)
             return $maxAge - $response->getAge();
         }
 
-        return null;
+        return;
```

Comment:
would prefer `return null;`, since it is more explicit. And my IDE is complaining about it ;-)